packages/amplify-category-api/src/provider-utils/awscloudformation/CfnApiArtifactHandler.ts

```diff
@@ -0,0 +1,176 @@
+import { ApiArtifactHandler } from '../ApiArtifactHandler';
```

Comment:
I think we should standardize naming of our files. We're using kebab-case for majority of our existing file names.

Reply:
I don't have super strong opinions here other than consistency, but it seems to be customary for files that define classes / interfaces to have the class name as the file name which is what I did here

Reply:
Discussed offline and file-names have been changed to kebab-case.